src/workspace.coffee

```diff
@@ -371,9 +371,11 @@ class Workspace extends Model
   Section: Opening
   ###
 
-  # Essential: Open a given a URI in Atom asynchronously.
-  #
-  # * `uri` A {String} containing a URI.
+  # Essential: Open a given URI in Atom asynchronously. 
+  # If no URI is given, or URI does not resolve to an existing file, 
```

Comment:
It was me agreeing to not documenting all the save-related behavior here.

About your initial suggesting: replacing

> or URI does not resolve to an existing file,

with

> or URI is not already open in an existing pane item

I'm not sure what's best.

Your suggestion states that if the uri is already open, it wan't open again, but is that true? Or is that dependent on how the open handler are implemented? (Cause in a small test project I have, I get multiple tabs with the same uri... ..not that I wanted to, but I can't find out how to prevent it...)

My suggestion states that if the uri is a file uri but the file doesn't exist, you will get a new text editor. 

So...  What about 


>  # Essential: Opens a given URI in Atom asynchronously, if it's not already open.
>  # If no URI is given, or URI is a filepath and the file does not exist, a new empty text editor is created.

or 

>  ..., or URI is to a file that does not exist, ...